about.html

```diff
@@ -0,0 +1,214 @@
+          <a href="#" id="close_nav" class="close_nav d-flex justify-content-end d-none">
+             <span class="material-symbols-outlined text-black">
+              close
+            </span></a>
+          <ul class="navbar-nav ml-auto d-flex flex-fill">
+            <li class="nav-item flex-fill my-auto home_item">
+              <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="nav-item flex-fill my-auto">
+              <a class="nav-link active-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item flex-fill my-auto">
+              <a class="nav-link" href="#program">Program</a>
+            </li>
+            <li class="nav-item flex-fill my-auto">
+              <a class="nav-link" href="#join">Join</a>
+            </li>
+            <li class="nav-item flex-fill my-auto">
+              <a class="nav-link" href="#speakers">Speakers</a>
+            </li>
+            <li class="nav-item flex-fill my-auto">
+              <a class="nav-link" href="#partners">Partners</a>
+            </li>
+            <li class="nav-item flex-fill active_compaign text-center">
+              <a class="nav-link" href="#">ES Compaign</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>  
+    <section class=" container-fluid my-auto about-header d-flex flex-column align-center justify-content-center text-center">
+        <div class="row">
+          <div class="col-md-12 col-xs-12 col-sm-12 my-auto mian-box">
+            <div class="upper-content my-auto py-5 ">
+              <div class="row justify-content-center">
+                <div class="col-md-6 col-sm-10 col-xs-12"> 
+                  <h1 class="fw-bold text-color">Educational Symposium <span class="text-uppercase">GLOBAL SUMMIT</span> 2023</h1>
+                </div>
+              </div>
+              <div class="row mt-5 justify-content-center">
+                <div class="col-md-8 col-sm-10 col-xs-12 description_div">
+                  <p class="font_size mt-3 fw-bold text-black ">An Educational Symposium is an event that brings together educators, researchers, administrators, and other professionals in the field of education to share knowledge, discuss innovative practices, and explore current trends and challenges in education. It provides a platform for collaboration, networking, and professional development.</p>
+              
+                </div>
+              </div>
+             
+             <div class="row mt-5 justify-content-center">
+              <div class="col-md-8 col-sm-10 col-xs-12">
+                <div class="lower-content">
+                  <p class="fw-bold font_size2">Please contact us per email for any further questions about Educational Symposium Global Summit 2023!</p>  
+                  <p class="fw-bold font_size3">educationalsymposium2023@gmail.com.</p>  
+                </div> 
+              </div>
+             </div> 
```

Comment:
- [ ] Kindly follow the given color of the text for your design as requested by the previous reviewer.

Your Design
![image](https://github.com/kamranasif506/Educational-Symposium/assets/81932478/e9c85be1-f846-4902-81c6-ee56eb7daaa8)

Given Design
![image](https://github.com/kamranasif506/Educational-Symposium/assets/81932478/94553626-3d27-48c8-9d16-5680c4d0d473)

### Note that the color swatch provided is not dark but rather a kind of faded grey
![image](https://github.com/kamranasif506/Educational-Symposium/assets/81932478/ac0c3b85-814f-4dd3-bd1e-2371810213f5)